scripts/auto-rebase/rebase.sh

```diff
@@ -405,6 +405,15 @@ update_manifests() {
     # 1) Adopt resource manifests
     #    Selectively copy in only those core manifests that MicroShift is already using
     cp "${STAGING_DIR}/cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/ns.yaml" "${REPOROOT}"/assets/core/0000_50_cluster-openshift-controller-manager_00_namespace.yaml
+    cp "${STAGING_DIR}/cluster-openshift-controller-manager-operator/bindata/v3.11.0/openshift-controller-manager/route-controller-informer-clusterrole.yaml" "${REPOROOT}"/assets/core/
```

Comment:
can we have the management of `ingress-to-route-controller-clusterrole.yaml` and `ingress-to-route-controller-clusterrolebinding.yaml` resources in the same place? And move also the initialization from the` openshift-router` to the route-controller-manager since it should be collocated there?

Reply:
There is a specific section for components in the rebase script. The openshift-router is handled here:

https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/rebase.sh#L491-L499

As for the initialization, it's also part of the service that starts the components such as `service-ca` deployment, `dns` deployment, etc.

https://github.com/openshift/microshift/blob/main/pkg/components/components.go#L23-L25

If we move the router outside this service, I believe it would look inconsistent.

Reply:
We should keep the router as is. I am just saying we should move the RBAC files for `ingress-to-route-controller` out. Since these files are only relevant for route-controller-manager and not for openshift-router.

Reply:
also, it would be good to have a comment above all these manifests from OCM-o that they belong to route-controller-manager

Reply:
Ok, understood! I'll move them together then. Thakns Filip!